modules/ROOT/pages/tgw-manage-arm.adoc

```diff
@@ -0,0 +1,101 @@
+= Manage Transit Gateway Attachments
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+image:logo-cloud-active.png[link="/runtime-manager/deployment-strategies#cloudhub", title="CloudHub"]
+image:logo-hybrid-disabled.png[link="/runtime-manager/deployment-strategies#hybrid", title="Hybrid"]
+image:logo-server-disabled.png[link="/runtime-manager/deployment-strategies#anypoint-platform-private-cloud-edition", title="Anypoint Platform PCE"]
+image:logo-rtf-disabled.png[link="/runtime-manager/deployment-strategies#anypoint-runtime-fabric", title="Runtime Fabric"]
+
+
+Use Runtime Manager to manage transit gateway attachments.
+
+== Rename a Transit Gateway Attachment
+
+To rename a transit gateway attachment:
+
+. On the *Transit Gateways* page, click the Transit Gateway menu (*...*) and select *Edit name*:
++
+_update screenshot - content and styles changed_
++
+image::tgw-rename.png[Edit name option in the Transit Gateways menu]
+. In the popup window, enter a new name and click *Save*.
+
+
+== Remove a Route from the Transit Gateway Route Table
+
+To remove a route:
+
+. On the *Transit Gateways* page, in the *VPCs* section, click the *X* icon in the row for a TGW route in the route table:
```

Comment:
This is a bit awkward: "click the *X* icon in the row for a TGW route in the route table"
Here's just one possibility:
```suggestion
. On the *Transit Gateways* page, in the *VPCs* section, click the *X* icon in the appropriate TGW route row of the route table:
```